get_stacktrace.sh

```
@@ -0,0 +1 @@
+./gradlew test --tests "com.sayai.record.fantasy.service.FantasyDraftServiceAutoPickTest.testAutoPick_FirstPickRule_Disabled" --info | grep -A 30 "NullPointerException"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)